src/views/Lottery/components/NextDrawCard.tsx

```diff
@@ -127,6 +127,16 @@ const NextDrawCard = () => {
     return ''
   }
 
+  let ticketRoundText
+  if (userTicketCount > 1) {
+    ticketRoundText = t('You have %amount% tickets this round', { amount: userTicketCount })
+  } else {
+    ticketRoundText = t('You have %amount% ticket this round', { amount: userTicketCount })
+  }
```

Comment:
We can use a ternary here instead.
`const ticketRoundText = userTicketCount > 1 ?...`

Reply:
Done